code/game/dna/genes/goon_disabilities.dm

```diff
@@ -38,21 +38,22 @@
 	deactivation_message = "You no longer feel awful and sick all over."
 	flags = GENE_UNNATURAL
 
-	New()
-		..()
-		block=RADBLOCK
+/datum/dna/gene/disability/radioactive/New()
+	..()
+	block=RADBLOCK
 
-	OnMobLife(var/mob/owner)
-		owner.radiation = max(owner.radiation, 20)
-		for(var/mob/living/L in range(1, owner))
-			if(L == owner)
-				continue
-			to_chat(L, "<span class='warning'>You are enveloped by a soft green glow emanating from [owner].</span>")
-			L.apply_radiation(5, RAD_INTERNAL)
-		return
+/datum/dna/gene/disability/radioactive/OnMobLife(var/mob/owner)
+	owner.radiation = max(owner.radiation, 20)
+	for(var/mob/living/L in range(1, owner))
+		if(L == owner)
+			continue
+		to_chat(L, "<span class='warning'>You are enveloped by a soft green glow emanating from [owner].</span>")
+		L.apply_radiation(5, RAD_INTERNAL)
```

Comment:
>RAD_INTERNAL

Why would they be receiving radiation from an internal source?

Use RAD_EXTERNAL so they can protect themselves using radioactive protection gear, and know if it's happening with the radiation detection badges.